src/tss2-tcti/tcti-tbs.c

```diff
@@ -0,0 +1,286 @@
+/* SPDX-License-Identifier: BSD-2 */
+/*
+ * Copyright (c) 2015 - 2018 Intel Corporation
+ * All rights reserved.
+ */
+
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <windows.h>
+#include <Tbs.h>
+
+#include "tss2_tcti.h"
+#include "tss2_tcti_tbs.h"
+
+#include "tcti-common.h"
+#include "tcti-tbs.h"
+#define LOGMODULE tcti
```

Comment:
Does this `#define` need to be before the `log.h` include? Perhaps move this onto it's own block of lines?